Monika After Story/game/script-brbs.rpy

```diff
@@ -15,13 +111,57 @@ init 10 python in mas_brbs:
         OUT:
             A randomly selected quip for coming back to the spaceroom
         """
+        return renpy.substitute(random.choice(WB_QUIPS_NORMAL))
 
-        return renpy.substitute(renpy.random.choice([
-            _("So, what else did you want to do today?"),
-            _("What else did you want to do today?"),
-            _("Is there anything else you wanted to do today?"),
-            _("What else should we do today?"),
-        ]))
+    def idle_setup(brb_label=None, brb_callback_label=None):
+        """
+        This callback is called in every brb event to enable idle mode
+
+        IN:
+            brb_label - the label of this brb event, if None, use the current label
+                (Default: None)
+            brb_callback_label - the callback label of this brb event, if None, we build it here
+                (Default: None)
+        """
+        # Get currect label
+        if brb_label is None:
+            brb_label = store.mas_submod_utils.current_label
+
+        # Add idle extra exps
+        store.mas_moni_idle_disp.add_by_tag("idle_mode_exps")
+
+        # Set vars
+        store.mas_globals.in_idle_mode = True
+        store.persistent._mas_in_idle_mode = True
+        store.persistent._mas_idle_data[brb_label] = True
```

Comment:
people seemed to have misunderstood what this var is for. `_mas_idle_data` is temporary persistent data storage so that the brb can store data for later use (more info in #3435). Think things like:

* changing the callback path based on dialogue in the brb start label
* changing crashed and reload greetings based on the brb that occurred 

I think people copied the test example (which just set the label key to True) without understanding what this is for. 

So:
1. this should be something the brb topics can pass in (and it should be any arbitrary data)
2. teardown shouldn't clear the entire db - maybe just the brb_label node is fine, but it should also be optional. I can imagine  a case where a brb wants to vary its dialogue if the user used it more than once in like the same session or something.

Reply:
Alright, this is actually cool.

Although, right now we also clear `persistent._mas_idle_data` entirely. I think we should call `teardown` in the callbacks, then. And if needed, we will clear the data for *this* brb in the callback. If a brb doesn't have a callback, we'll call a generalised version which will call `teardown` with the default params.